src/main/java/gov/cabinetoffice/gap/applybackend/service/SubmissionService.java

```diff
@@ -70,7 +64,11 @@ public Submission getSubmissionFromDatabaseBySubmissionId(final String userId, f
                 .orElseThrow(() -> new NotFoundException(
                         String.format("No Submission with ID %s was found", submissionId)));
 
-        populateEssentialInformation(userId, submission);
+        // Don't love this. Begging for exceptions to be thrown.
+        if (submission.getApplication().getGrantScheme().getVersion() == 1) {
+            populateEssentialInformation(userId, submission);
+        }
```

Comment:
In fairness, if this did somehow throw an exception here thats probably correct as it would mean something in that chain is missing and that would certainly be an unrecoverable error which would break many things in the app.